src/sinks/util/buffer/vec.rs

```diff
@@ -21,6 +21,16 @@ impl<T> Batch for VecBuffer<T> {
     type Input = T;
     type Output = Vec<T>;
 
+    fn get_settings_defaults(
```

Comment:
I see this fn interface enough times I wonder if it should be its own trait?

Reply:
It's an interesting idea, though I think out of scope for this PR, unless you are seeing it in different places in this set of interfaces.

Reply:
Agreed!